be2-scala/src/main/scala/ch/epfl/pop/Server.scala

```diff
@@ -25,37 +25,34 @@ object Server {
     val PORT = 8000
     val PATH = ""
 
-    val root = Behaviors.setup[Nothing] { context =>
-      implicit val system: ActorSystem[Nothing] = context.system
+    val system = akka.actor.ActorSystem("magic")
```

Comment:
We should probably use another string :)

Reply:
Done